packages/sdk/src/sdk-blockchains/solana/common/auction-house.ts

```diff
@@ -15,14 +15,15 @@ export function getAuctionHouse(currency: CurrencyType, auctionHouseMapping?: So
 		if (auctionHouseMapping && auctionHouseMapping["SOLANA_SOL"]) {
 			return toPublicKey(auctionHouseMapping["SOLANA_SOL"].address)
 		}
+		return toPublicKey("raria47jXd4tdW6Dj7T64mgahwTjMsVaDwFxMHt9Jbp")
```

Comment:
Are you sure that address should be here?

Reply:
Yes, that default rarible ah